prwkv/rwkvrnnmodel.py

```diff
@@ -83,8 +83,8 @@ def clear_memory(self):
         self.init_state = None
         self.init_logits = None 
         
-    def warmup_with_context(self,context:List[int]):
-        init_state = None 
+    def warmup_with_context(self,context:List[int],clear=True):
```

Comment:
is this meant to allow to build up the context further?

Reply:
yeah; when the setup is all together you can process context with the GPT form and then generate with the RNN form. but maybe it is not the interface you prefer.